packages/vite-plugin-rune/src/terser.ts

```diff
@@ -46,7 +46,10 @@ export function terserPlugin(): Plugin {
 
       await Promise.all(
         Object.keys(bundle).map(async (name) => {
-          if (!shouldMinify || name === "logic.js") {
+          if (
+            !shouldMinify ||
+            (name === "logic.js" && !process.env.RUNE_MINIFY_LOGIC)
```

Comment:
```suggestion
            (name === "logic.js" && process.env.RUNE_MINIFY_LOGIC !== "0")
```

Would be great to make `RUNE_MINIFY_LOGIC = 0` still minify the logic.js file :)